examples/nexrad/nexrad_copy.py

```diff
@@ -0,0 +1,160 @@
+from siphon.radarserver import RadarServer
+from siphon.cdmr import Dataset
+
+# Example of CopySQLClient usage to stream data from NEXRAD Level 2
+# data to CARTO.
+#
+# For more information see the following URL's:
+#   - https://carto.com/blog/mapping-nexrad-radar-data/
+#   - https://gist.github.com/stuartlynn/a7868cf8ca02931a6408
+#   - http://nbviewer.jupyter.org/gist/dopplershift/356f2e14832e9b676207
+
+# Set up our own logger after cleaning up siphon's one
+logger = logging.getLogger()
+list(map(logger.removeHandler, logger.handlers[:]))
+ch = logging.StreamHandler()
+formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+ch.setFormatter(formatter)
+logger.addHandler(ch)
+logger.setLevel(logging.INFO)
+
+# Parse input arguments
+parser = argparse.ArgumentParser(description=(
+    'Example of CopySQLClient usage to stream data from NEXRAD Level 2'
+    ' data to CARTO.'
+))
+
+parser.add_argument('--base_url', type=str, dest='CARTO_BASE_URL',
+                    default=os.environ.get('CARTO_API_URL', ''),
+                    help=('Set the base URL. For example:'
+                          ' https://username.carto.com/'
+                          ' (defaults to env variable CARTO_API_URL)'))
+
+parser.add_argument('--api_key', dest='CARTO_API_KEY',
+                    default=os.environ.get('CARTO_API_KEY', ''),
+                    help=('Api key of the account'
+                          ' (defaults to env variable CARTO_API_KEY)'))
+
+args = parser.parse_args()
+
+if not args.CARTO_BASE_URL or not args.CARTO_API_KEY:
+    sys.exit(parser.print_usage())
+
+auth_client = APIKeyAuthClient(args.CARTO_BASE_URL, args.CARTO_API_KEY)
+sql_client = SQLClient(auth_client)
+copy_client = CopySQLClient(auth_client)
+
+# Create a table suitable to receive the data
+logger.info('Creating table nexrad_copy_example...')
+sql_client.send("""CREATE TABLE IF NOT EXISTS nexrad_copy_example (
+  the_geom geometry(Geometry,4326),
+  reflectivity numeric
+)""")
+sql_client.send(
+    "SELECT CDB_CartodbfyTable(current_schema, 'nexrad_copy_example')")
+logger.info('Done')
+
+logger.info('Trying to connect to the THREDDS radar query service')
+rs = RadarServer(
+    'http://thredds.ucar.edu/thredds/radarServer/nexrad/level2/IDD/')
+
+logger.info('Quering data from the station')
+query = rs.query()
+query.stations('KLVX').time(datetime.utcnow())
+assert rs.validate_query(query)
+
+catalog = rs.get_catalog(query)
+logger.info('Avaliable datasets: %s' % catalog.datasets)
+logger.info('Using the first one')
+ds = list(catalog.datasets.values())[0]
+data = Dataset(ds.access_urls['CdmRemote'])
+logger.info('Got the following data: %s' % data.Title)
+logger.info(data.Summary)
+
+
+# A helper method to clean up the data
+def raw_to_masked_float(var, data):
+    # Values come back signed. If the _Unsigned attribute is set, we
+    # need to convert from the range [-127, 128] to [0, 255].
+    if var._Unsigned:
+        data = data & 255
```

Comment:
oh, I've just learned this is equivalent to data += 256 in Python 😮

Reply:
Copied from the jupyter notebook referenced above, that's a bitwise AND operation with a bit mask, isn't it? Maybe I can rewrite it as
```python
data = data & 0xFF
```

for clarity's sake.